ext/SpeedyWeatherGeoMakieExt/SpeedyWeatherGeoMakieExt.jl

```diff
@@ -16,49 +16,62 @@ cell centers and faces. Optionally, add coastlines and a background image of the
 function SpeedyWeather.globe(
     Grid::Type{<:AbstractGridArray},
     nlat_half::Integer;
+    interactive::Bool = true,
+    title::String = "$(RingGrids.get_nlat(typeof(grid), grid.nlat_half))-ring $(RingGrids.horizontal_grid_type(grid))",
     color = :black,
     faces::Bool = true,
     centers::Bool = true,
     coastlines::Bool = true,
     background::Bool = true,
 )
-    transf = GeoMakie.Geodesy.ECEFfromLLA(GeoMakie.Geodesy.WGS84())
 
     fig = Figure(size=(800, 800));
-    ax = LScene(fig[1,1], show_axis=false);
+
+    if interactive
+        transf = GeoMakie.Geodesy.ECEFfromLLA(GeoMakie.Geodesy.WGS84())
+        ax = LScene(fig[1,1], show_axis=false);
+    else
+        ax = GeoAxis(fig[1, 1],
+            title= title,
+            dest = "+proj=ortho +lon_0=30 +lat_0=45")
+    end
```

Comment:
Ideally this shouldn't be necessary...I have figured out why the polygons are not triangulating correctly (I think).